### ISSUE TITLE
ai85net20 model training

### ISSUE THREAD

**Issue by smtalds** [MaximIntegratedAI/ai8x-training#214]
I want to use ai85net20 model and  i changed dimension on train file and model file (dimension = 128,128). They are not match with this model. I set dimension 0 and 1 index and i set num_channels  dimension[2]. Then i got this error:
" Expected 4-dimensional input for 4-dimensional weight [15, 1, 3, 3], but got 3-dimensional input of size [1, 128, 128] instead". Why dimension is not match ?

**Comment by smtalds** [comment]
I fix dimension problem i guess, i changed input  size (1,128,128 )on kws20.py dataset file and i changed again on train.py dimension like default. Now my loss function is started 3.0 and fit 1.62 but validation and test accuracy is %92. (for top5, top1 is %70). Is kws untrainable for this model ?

**Comment by aniktash** [comment]
The ai85net20 model is an earlier implementation, developed to have the MFCC of the audio as the input rather than raw data. The model may not perform with raw data. The dataloader needs some updates to train with the MFCC.  I'd suggest using ai85kws20netv3 or its variations which work with raw data and changing the dataset if you need different keywords.

**Comment by smtalds** [comment]
Thank you so much.
